SPECS/blobfuse2/blobfuse2.spec

```diff
@@ -80,11 +80,14 @@ install -D -m 0644 ./setup/blobfuse2-logrotate %{buildroot}%{_sysconfdir}/logrot
 %{_sysconfdir}/logrotate.d/blobfuse2
 
 %changelog
+* Fri Nov 17 2023 Anubhuti Shruti <ashruti@microsoft.com> - 2.1.2-1
+- Bump version to 2.1.2
+
 * Thu Nov 02 2023 Sourav Gupta <souravgupta@microsoft.com> - 2.1.1-1
 - Bump version to 2.1.1
 
 * Mon Oct 16 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 2.1.0-3
-- Bump release to rebuild with go 1.20.9
+- Bump release to rebuild with go 1.20.10
```

Comment:
NIT: why are we changing this ?

Reply:
Hey, thanks for pointing out. I didn't make this change intentionally, not sure why it came up. Changed it back to original.